Makefile

```diff
@@ -110,14 +108,14 @@ test-lint:
 
 .PHONY: test-unit
 test-unit:
-	$(GO) test $(GOFLAGS) -cover -run $(TESTS) $(PKG) $(TESTFLAGS)
```

Comment:
how come we're removing the `-cover` and `-run` flags?

Reply:
hmm, I'm not 100% sure why I removed them. I think it was because I noticed a significant performance improvement while testing, but that's irrelevant to this PR. I'll add them back in and do some more digging.

Reply:
Looks like there have been performance improvements made to Go 1.9 since this PR was initially introduced. I've reverted this.